.github/workflows/update-flake-lock.yml

```
@@ -0,0 +1,66 @@
+        run: nix flake check
+
+      - name: Check Template Flake
+        id: check_template
+        continue-on-error: true
+        run: |
+          cd template
+          sed -i 's|url = "github:richen604/hydenix"|url = "path:../"|' flake.nix
+          nix flake check
+          git checkout flake.nix
+          rm -f flake.lock
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)